src/plasmapy/utils/data/downloader.py

```diff
@@ -214,7 +214,7 @@ def _update_repo_blob_dict(self) -> None:
             warnings.warn(
                 "URL did not return the expected JSON file: "
                 f"{self._API_BASE_URL}. "
-                f"Response content: {reply.content}. Exception: {err}"
+                f"Response content: {reply.content.decode('utf-8')}. Exception: {err}"
```

Comment:
This is completely unrelated to this PR, but it popped up in the errors and looked like a simple fix so I just made it...